<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsOgrSourceSelectBase</class>
 <widget class="QDialog" name="QgsOgrSourceSelectBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Vector Layer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="srcGroupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Source Type</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="radioSrcFile">
            <property name="text">
             <string>F&amp;ile</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioSrcDirectory">
            <property name="text">
             <string>&amp;Directory</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioSrcDatabase">
            <property name="text">
             <string>Da&amp;tabase</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioSrcProtocol">
            <property name="text">
             <string>Protoco&amp;l: HTTP(S), cloud, etc.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioSrcOgcApi">
            <property name="text">
             <string>OGC API</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Encoding</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbEncodings">
            <property name="minimumSize">
             <size>
              <width>341</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="protocolGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Protocol</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="1">
         <widget class="QLineEdit" name="mKey"/>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cmbProtocolTypes"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelKey">
          <property name="text">
           <string>Object key</string>
          </property>
          <property name="buddy">
           <cstring>mKey</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="protocolURI"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelBucket">
          <property name="text">
           <string>Bucket or container</string>
          </property>
          <property name="buddy">
           <cstring>mBucket</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="mBucket"/>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QGroupBox" name="mAuthGroupBox">
          <property name="title">
           <string>Authentication</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QgsAuthSettingsWidget" name="mAuthSettingsProtocol" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelProtocolURI">
          <property name="text">
           <string>&amp;URI</string>
          </property>
          <property name="buddy">
           <cstring>protocolURI</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="fileGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Source</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="labelDirectoryType">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cmbDirectoryTypes">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelSrcDataset">
          <property name="text">
           <string>Vector Dataset(s)</string>
          </property>
          <property name="buddy">
           <cstring>mFileWidget</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QgsFileWidget" name="mFileWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="dbGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Database</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cmbDatabaseTypes">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Connections</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <widget class="QPushButton" name="btnNew">
             <property name="text">
              <string>New</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btnEdit">
             <property name="text">
              <string>Edit</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="btnDelete">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="3">
            <widget class="QComboBox" name="cmbConnections"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QgsCollapsibleGroupBox" name="mCredentialOptionsGroupBox">
       <property name="title">
        <string>Credential Options</string>
       </property>
       <layout class="QVBoxLayout" name="openOptionsVBoxLayout_2" stretch="1">
        <item>
         <layout class="QFormLayout" name="mCredentialOptionsLayout"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QgsScrollArea" name="scrollArea">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <property name="leftMargin" stdset="0">
        <number>0</number>
       </property>
       <property name="topMargin" stdset="0">
        <number>0</number>
       </property>
       <property name="rightMargin" stdset="0">
        <number>0</number>
       </property>
       <property name="bottomMargin" stdset="0">
        <number>0</number>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>529</width>
          <height>77</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QgsCollapsibleGroupBox" name="mOpenOptionsGroupBox">
           <property name="title">
            <string>Options</string>
           </property>
           <layout class="QVBoxLayout" name="openOptionsVBoxLayout">
            <item>
             <widget class="QLabel" name="mOpenOptionsLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QFormLayout" name="mOpenOptionsLayout">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Help</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qgsscrollarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsAuthSettingsWidget</class>
   <extends>QWidget</extends>
   <header>qgsauthsettingswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>radioSrcFile</tabstop>
  <tabstop>radioSrcDirectory</tabstop>
  <tabstop>radioSrcDatabase</tabstop>
  <tabstop>radioSrcProtocol</tabstop>
  <tabstop>cmbEncodings</tabstop>
  <tabstop>cmbProtocolTypes</tabstop>
  <tabstop>protocolURI</tabstop>
  <tabstop>mBucket</tabstop>
  <tabstop>mKey</tabstop>
  <tabstop>cmbDirectoryTypes</tabstop>
  <tabstop>cmbDatabaseTypes</tabstop>
  <tabstop>cmbConnections</tabstop>
  <tabstop>btnNew</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnDelete</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radioSrcDatabase</sender>
   <signal>toggled(bool)</signal>
   <receiver>dbGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
